### DISCUSSION TITLE
Minor Cleanup Opportunity: ```ExceptionDispatchInfo.Capture(ex).Throw()``` => ```ExceptionDispatchInfo.Throw(ex)```

### DISCUSSION THREAD

**Discussion by TonyValenti** [dotnet/runtime#105755] [Ideas] [answered]
I just noticed that there is an ```ExceptionDispatchInfo.Throw(ex)``` method which is equivalent to ```ExceptionDispatchInfo.Capture(ex).Throw()```.

There are a number of places in the runtime that do ```ExceptionDispatchInfo.Capture(ex).Throw()```.  These should be refactored to use ```ExceptionDispatchInfo.Throw(ex)```

**Comment by stephentoub** [comment, marked as answer]
I expect most of the ones you're seeing are in files that also compile for netstandard2.0.
